### DISCUSSION TITLE
Joins, include and select

### DISCUSSION THREAD

**Discussion by rifay-next99** [medz/dart-orm#339] [Q&A]
Im using orm in dart_frog. i want to join and. select particular data. in this prisma client these method is not found. but while using in nodejs is present why?

**Comment by medz** [comment]
@rifay-next99 select and include are only supported in version 4.0. You should use version 4.0.0-beta. (The official version will be released soon, and two bugs will be fixed before it is released). I wish you happy.

**Comment by rifay-next99** [comment]
What about the joins. how to join and  get the data from two toble. if i use rawQuery is givs only string format

**Comment by medz** [comment]
@rifay-next99 You should use the Fluent API to get relational data (in 3.x versions)

> Version 4.0 has just been officially released, which allows you to use Prisma's native methods to obtain relationships and retrieve relationship data
